src/Lykke.Service.ConfirmationCodes.AzureRepositories/AutofacRepositoriesModule.cs

```diff
@@ -69,6 +76,55 @@ protected override void Load(ContainerBuilder builder)
             builder.Register<ICallTimeLimitsRepository>(y =>
                 new CallTimeLimitsRepository(
                     AzureTableStorage<ApiCallHistoryRecord>.Create(_logsConnString, TableNameApiCalls, y.Resolve<ILogFactory>())));
+            
+            if (!string.IsNullOrEmpty(Environment.GetEnvironmentVariable("EncryptionKey")))
+            {
+                builder
+                    .Register(x =>
+                    {
+                        var manager = new EncryptedStorageManager(AzureTableStorage<EncryptionInitModel>.Create(
+                            _google2faConnString,
+                            TableNameGoogle2Fa,
+                            x.Resolve<ILogFactory>()));
+                        
+                        if (!manager.TrySetKey(Environment.GetEnvironmentVariable("EncryptionKey"), out string error))
+                        {
+                            var exception = new InvalidOperationException("EncryptionKey is not set");
+                            x.Resolve<ILogFactory>().CreateLog(this).WriteFatalError("SetEncryptionKey", error, exception);
+                            throw exception;
+                        }
+    
+                        return manager;
+                    })
+                    .As<EncryptedStorageManager>()
+                    .AutoActivate();
+            }
+            else
```

Comment:
The requirement for a production - to **have** "EncryptionKey" as environment variable. Let's at least write a warning here, to decrease risk of missing this key.